build-aux/m4/bitcoin_qt.m4

```diff
@@ -428,7 +432,7 @@ dnl Outputs: have_qt_test and have_qt_dbus are set (if applicable) to yes|no.
 AC_DEFUN([_BITCOIN_QT_FIND_LIBS_WITH_PKGCONFIG],[
   m4_ifdef([PKG_CHECK_MODULES],[
     QT_LIB_PREFIX=Qt5
-    qt5_modules="Qt5Core Qt5Gui Qt5Network Qt5Widgets"
+    qt5_modules="Qt5Core Qt5Gui Qt5Network Qt5Widgets Qt5Svg"
```

Comment:
I'd suggest not adding `Qt5Svg` here.
On my system (debian without Qt5Svg) removing this leads configure (with `--with-gui`) to fail with `configure: error: qtsvg not found` while with this the error message is more generic: `configure: error: Qt dependencies not found`.

Reply:
qt5svg is mandatory now. Have updated the dependencies doc with it.

Explanation: 
Our wallet is fully responsive now, every icon and drawable resource can be resized depending on the screen density (xml scalable vectors that we can expand or minimize programmatically, SVG resources). 

Previous wallet used only png resources, which were blocking any good quality screen resize. png force the wallet to have one icon/image file per screen density, instead of programmatically resize the same unique file, or it will get pixelated and decrease its quality a lot.

In linux qt5svg is supported in every distribution that i have tried, you just need to download `libqt5svg5-dev`.

Reply:
Yes but this lines make it so that the configure script doesn't tell you that qtsvg is missing.

Reply:
hmm ok, it's validated in other place but I'm not so sure if we can remove it from there, if qt5_modules is a global variable used in some other places then it should be checked before that line and not be removed. Will check it.

Still, i wouldn't spend too much time around this. People will always first read the build-xxx.md doc, which states the svg dependency anyway.

Reply:
`s/always/eventually` ;)

Reply:
Qt5Svg is indeed checked elsewhere and shouldn't be included in this particular line, who's purpose is to only check the top level Core Qt modules (those available/provided by the Qt5Base package).

So here, the Core modules are checked first, then (if present), additional modules and supporting libraries are checked elsewhere.

Reply:
Cannot argue much against our build system champ, will remove it then 👍 .